src/router/index.js

```diff
@@ -19,8 +19,8 @@ export default new Router({
     { path: '/top/:page(\\d+)?', component: TopListView },
     { path: '/new/:page(\\d+)?', component: NewListView },
     { path: '/recommendations/:page(\\d+)?', component: RecomendationListView },
-    { path: '/item/:id([A-Za-z0-9-_]+)?', component: ItemView },
-    { path: '/login', component: Login },
-    { path: '/regsiter', component: Register }
+    { path: '/post/:id([A-Za-z0-9-_]+)?', component: PostDetailsView },
+    { path: '/login', component: LoginView },
+    { path: '/regsiter', component: RegisterView }
```

Comment:
Can we refactor the "regsiter" route to be "register"?

Reply:
Or ship this as is and add it as issue (I know "regsiter" has been in there for awhile)